spec/unit/views/pages/form_spec.rb

```diff
@@ -16,7 +16,7 @@
     end
 
     let(:arbre_context) do
-      OpenStruct.new(params: params, helpers: helpers, assigns: {})
+      instance_double(Arbre::Context, helpers: helpers, assigns: {})
```

Comment:
@javierjulio does this work?

Reply:
Yes, I think so since tests run fine. Thanks!